libmultilabel/nn/networks/modules.py

```diff
@@ -0,0 +1,176 @@
+from abc import ABC, abstractmethod
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn.utils.rnn import pack_padded_sequence, pad_packed_sequence
+
+
+class Embedding(nn.Module):
+    """Embedding layer with dropout
+
+    Args:
+        embed_vecs (FloatTensor): The pre-trained word vectors of shape (vocab_size, embed_dim).
+        dropout (float): The dropout rate of the word embedding. Defaults to 0.2.
+    """
+
+    def __init__(self, embed_vecs, dropout=0.2):
+        super(Embedding, self).__init__()
+        self.embedding = nn.Embedding.from_pretrained(
+            embed_vecs, freeze=False, padding_idx=0)
+        self.dropout = nn.Dropout(dropout)
+
+    def forward(self, inputs):
+        return self.dropout(self.embedding(inputs))
+
+
+class RNNEncoder(ABC, nn.Module):
+    """Base class of RNN encoder with dropout
+
+    Args:
+        input_size (int): The number of expected features in the input.
+        hidden_size (int): The number of features in the hidden state.
+        num_layers (int): Number of recurrent layers.
```

Comment:
"Number of .." different from "The number of" of the previous two arguments. Same issue in other class as well

Reply:
It seems we don't have a consistent rule for this kind of comment style yet.
Maybe discuss on Mon and revise all of them in the next PR?